src/skins/vector/views/molecules/ServerConfig.js

```diff
@@ -20,37 +20,142 @@ var React = require('react');
 var Modal = require('matrix-react-sdk/lib/Modal');
```

Comment:
Will move to components directory.

>  how come PasswordLogin & CasLogin live in react-sdk but this lives in vector?

There's nothing intrinsic to Vector about Password/CAS login which I why I figured they should live in the React SDK. The text and arrangement of `ServerConfig` though feels very Vector-specific (it even specifically mentions Vector!) which is why I felt it should be in Vector.